### PR TITLE
Issue #41 Add option to use local time zone instead of UTC in audit logging

### PR DESCRIPTION
**Pull request by kurotsu**
## Solution
Add an item to the audit log service so that it can be switched to local time.

## Testing
Execute authentication and setting change, and refer to the audit log file (access.csv,activity.csv,authentication.csv,config.csv)

### CODE REVIEW

**Comment by ogis-miyamura** [comment]
Works well.
Could you add a test case for isLtzEnabled is 'true' ?

**Comment by kurotsu** [comment]
Commit reflecting the review findings.

**Comment by kurotsu** [comment]
> Works well.
> Could you add a test case for isLtzEnabled is 'true' ?

I can do it.

**Comment by kurotsu** [comment]
Update Copyright & Formatting Code.

File: openam-core/src/main/java/com/iplanet/services/comm/server/PLLAuditor.java [hunk 1]
```diff
@@ -86,7 +87,7 @@ public void auditAccessAttempt() {
 
             AuditEvent auditEvent = auditEventFactory.accessEvent(realm)
                     .forHttpServletRequest(httpServletRequest)
-                    .timestamp(startTime)
+                    .timestamp(startTime,auditEventPublisher.isLtzEnabled())
```
Comment (tsujiguchitky) [inline openam-core/src/main/java/com/iplanet/services/comm/server/PLLAuditor.java]: This is a minute thing, but add a space after the comma please.

File: openam-audit/openam-audit-core/src/test/java/org/forgerock/openam/audit/AMAccessAuditEventBuilderTest.java [hunk 2]
```diff
@@ -73,6 +74,25 @@ public void canBuildAccessAuditEventWithContext() throws Exception {
         assertJsonValue(accessEvent.getValue(), "/access-event.json");
     }
 
+    @Test
+    public void canBuildAccessAuditEventWithContextLtz() throws Exception {
```
[the remark this answers is not in the document]
Comment (ogis-miyamura) [reply openam-audit/openam-audit-core/src/test/java/org/forgerock/openam/audit/AMAccessAuditEventBuilderTest.java]: Thank you for adding the test.
However, the test did not pass in my build environment (UTC time zone).
I sent an small patch.

[the remark this answers is not in the document]
Comment (kurotsu) [reply openam-audit/openam-audit-core/src/test/java/org/forgerock/openam/audit/AMAccessAuditEventBuilderTest.java]: Thanks for your response.

File: openam-core/src/main/java/com/iplanet/services/comm/server/PLLAuditor.java (hunk 1 above)
Comment (kurotsu) [reply openam-core/src/main/java/com/iplanet/services/comm/server/PLLAuditor.java]: Done.

**Comment by tsujiguchitky** [comment]
LGTM

**Pull request merged**